language/diem-framework/modules/PaymentScripts.move

```diff
@@ -133,27 +133,27 @@ module DiemFramework::PaymentScripts {
     }
 
     spec peer_to_peer_with_metadata {
-        include PeerToPeer<Currency>;
+        include PeerToPeer<Currency>{payer_signer: payer};
         include DualAttestation::AssertPaymentOkAbortsIf<Currency>{
             payer: Signer::spec_address_of(payer),
             value: amount
         };
     }
 
     spec peer_to_peer_by_signers {
-        include PeerToPeer<Currency>{payee: Signer::spec_address_of(payee)};
+        include PeerToPeer<Currency>{payer_signer: payer, payee: Signer::spec_address_of(payee)};
     }
 
     spec schema PeerToPeer<Currency> {
         use Std::Errors;
 
-        payer: signer;
+        payer_signer: signer;
```

Comment:
Changed the name to bypass [the bug](https://github.com/diem/diem/issues/8854) mentioned in the description of this PR.